pkg/client/dispatcher.go

```diff
@@ -243,15 +243,14 @@ func (a *actionListenerImpl) Actions(ctx context.Context, errCh chan<- error) (<
 
 					if err != nil {
 						a.l.Error().Msgf("Failed to subscribe: %v", err)
-						errCh <- fmt.Errorf("failed to subscribe: %w", err)
-						return
+						panic(fmt.Errorf("failed to subscribe: %w", err))
```

Comment:
is this fine?

Reply:
Yes, this is after a retrier so makes sense.